README.md

```diff
@@ -1,2 +1,1113 @@
+        }
+    }
+```
+
+Note that multiple `item` elements appear inside of the `string-array`
+element in the example XML. The `localizableType` assigns the text value
+of each item to the "source" field in the resource. In this way, each one
+of them is added as an entry in the array in the order they appeared
+in the original XML file.
+
+#### Plural Resources
+
+When the `localizable` keyword is given for an object type, that object
+may encode a plural string. Plurals are not possible if the JSON schema
+for the current element is a primitive type like string or number. 
+The `localizableType` keyword should contain a type property with
+the value "plural" to specify that.
+
+In order to fully specify a plural, elements or attributes under that
+main element must contain a string for various plural categories.
+To specify which category to use, the `localizableType` keyword contains a
+`category` property.
+
+The list of allowed plural categories are defined in the
+[Unicode plural rules](http://cldr.unicode.org/index/cldr-spec/plural-rules):
+
+- zero
+- one
+- two
+- few
+- many
+- other
+
+The "one" and the "other" property are required for source files in English.
+Other languages will have different combinations of plural categories.
+
+Example XML:
+
+```xml
+    <string-plural id="unique">
+        <singular>This is the singular string</singular>
+        <plural>This is the plural string</plural>
+    </string-plural>
+```
+
+Example JSON schema file for the above XML:
+
+```json
+        "string-plural": {
+            "type": "object",
+            "localizable": true,
+            "localizableType": "plural",
+            "properties": {
+                "_attributes": {
+                    "type": "object",
+                    "properties": {
+                        "id": {
+                            "type": "string",
+                            "localizableType": {
+                                "key": "_value"
+                            }
+                        }
+                    }
+                },
+                "singular": {
+                    "type": "object",
+                    "properties": {
+                        "_text": {
+                            "type": "string",
+                            "localizableType": {
+                                "source": "_value",
+                                "category": "one"
+                            }
+                        }
+                    }
+                },
+                "plural": {
+                    "type": "object",
+                    "properties": {
+                        "_text": {
+                            "type": "string",
+                            "localizableType": {
+                                "source": "_value",
+                                "category": "other"
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
```

Comment:
Remove the last two hanging braces here as they were not opened.

Reply:
removed